### ISSUE TITLE
[1] 1.3 Adaptable: Each form must have labels

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/utk-hyku#40]
In addition to this ticket, please create an issue for this in [Hyku](https://github.com/samvera/hyku/issues) and link it to this ticket.

# Summary

(Note from Shana - I am not 100% what this failure relates to. Is it saying that each checkbox seen under the select column should have a label?) 

Please confirm with Diem

# Acceptance Criteria

- [ ] Each form (checkbox) must have labels.

# Screenshots or Video

![Image](https://user-images.githubusercontent.com/29311858/203177224-3ee94222-d2e3-4ae3-8689-07e399d479a1.png)

# Testing Instructions
1. Login to staging as an admin user
1. Visit /dashboard/my/collections?locale=en, observe that the check boxes have labels

# Notes

This ticket has been created as a result of an accessibility audit, section (1.3.1 Info and Relationships): ref: [Report for https___test.hykucommons.org_.pdf](https://github.com/scientist-softserv/utk-hyku/files/10061262/Report.for.https___test.hykucommons.org_.pdf)

**Results for:** (https://test.hykucommons.org/dashboard/my/collections?locale=en)

**Outcome:** Failed

**Findings:** Each form (the 8 checkboxes) must have labels.

**Further reading:** https://dequeuniversity.com/rules/axe/4.3/label?application=AxeChrome
(https://dequeuniversity.com/rules/axe/4.3/label?application=AxeChrome

**Comment by DiemBTran** [comment]
Linked items:

- EPIC: Accessibility Improvements

**Comment by DiemBTran** [comment]
Kirk June 2022

---

@DiemBTran Is this ticket pertaining to adding labels to each checkbox?

--

Diem June 2022

---

Yes, this is pertaining to adding labels to each checkbox... I now realize that there are indeed labels for each checkbox. However, they say `Check to add to a collection or edit list`, but they should additionally describe the `Delete Selected` option as well.

---

Diem June 2022

---



![Image](https://user-images.githubusercontent.com/29311858/203177502-2b52c9ad-9cf5-44cc-9551-7c6b8b8588f3.png)

**Comment by jillpe** [comment]
related to #387

**Comment by ShanaLMoore** [comment]
related 
- https://github.com/orgs/scientist-softserv/projects/43/views/1?filterQuery=contribute+&pane=issue&itemId=34071329

**Comment by jillpe** [comment]
SoftServ QA: PASS!

Each of the 8 forms had a label, however it says `Check to batch delete collections`. It did not describe the `Check to add to a collection or edit list` actions as well because this page does not give opportunity for those actions.

![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/552723dd-a238-4cc0-bfa0-e5b015b9a2c2)
